ansible/playbooks/ptf_installation.yaml

```diff
@@ -5,21 +5,43 @@
   become: true
   become_user: root
   vars:
-    ptf_url: "your_ptf_url"
-    ptf_user: "your_username_here"
-    ptf_password: "your_pass_here"
+    url_timeout: 30
+    url_retries_cnt: 5
+    url_retries_delay: 10
     ptf_dir: "/tmp/ptf_dir"
+  
   tasks:
+    - name: Include external variables
+      ansible.builtin.include_vars: ./vars/hana_media.yaml
+
     - name: Create directory for PTF installation
       file:
         path: "{{ ptf_dir }}"
         state: directory
         mode: '0755'
 
-    - name: Download PTF files recursively with wget
-      command: "wget --no-directories --recursive --reject 'index.html*' --user={{ ptf_user }} --password={{ ptf_password }} --no-parent {{ ptf_url }}"
-      args:
-        chdir: "{{ ptf_dir }}"
+    - name: Convert comma-separated string to list
+      set_fact:
+        az_blobs: "{{ ptf_files | split(',') }}"
+
+    - name: Extract the first part of az_container_name
+      set_fact:
+        az_container_top: "{{ az_container_name.split('/')[0] }}"
```

Comment:
we can avoid this "trick" (that create hidden dependency of path used in hana_media.yaml and playbooks/sap-hana-download-media.yaml) and just force the user to provide the path in a second variable just for this playbook

```
ptf_installation.yaml -e ptf_container="aaa/bbb/ptf" -e ptf_files="aaa.rpm,bbb.rpm"
```

or with the path in each file element
```
ptf_installation.yaml  -e ptf_files="aaa/bbb/ptf/aaa.rpm,aaa/bbb/ptf/bbb.rpm"
```

Reply:
We can decide to use a dedicated container for PTF.
Maybe one with a subfolder for each PTF